[2024-02-15T15:42:47.641+0900] {processor.py:161} INFO - Started process (PID=1044) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
[2024-02-15T15:42:47.642+0900] {processor.py:830} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-02-15T15:42:47.643+0900] {logging_mixin.py:188} INFO - [2024-02-15T15:42:47.643+0900] {dagbag.py:538} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
[2024-02-15T15:42:47.661+0900] {logging_mixin.py:188} INFO - [2024-02-15T15:42:47.657+0900] {dagbag.py:348} ERROR - Failed to import: /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 584, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-02-15T15:42:47.662+0900] {processor.py:842} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
[2024-02-15T15:42:47.681+0900] {processor.py:183} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py took 0.044 seconds
[2024-02-15T15:43:44.449+0900] {processor.py:161} INFO - Started process (PID=1111) to work on /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
[2024-02-15T15:43:44.450+0900] {processor.py:830} INFO - Processing file /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py for tasks to queue
[2024-02-15T15:43:44.452+0900] {logging_mixin.py:188} INFO - [2024-02-15T15:43:44.451+0900] {dagbag.py:538} INFO - Filling up the DagBag from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
[2024-02-15T15:43:44.467+0900] {logging_mixin.py:188} INFO - [2024-02-15T15:43:44.466+0900] {dagbag.py:348} ERROR - Failed to import: /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py", line 138, in <module>
    branching_venv = BranchPythonVirtualenvOperator(
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 437, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 584, in __init__
    raise AirflowException("PythonVirtualenvOperator requires virtualenv, please install it.")
airflow.exceptions.AirflowException: PythonVirtualenvOperator requires virtualenv, please install it.
[2024-02-15T15:43:44.468+0900] {processor.py:842} WARNING - No viable dags retrieved from /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py
[2024-02-15T15:43:44.488+0900] {processor.py:183} INFO - Processing /usr/local/lib/python3.10/dist-packages/airflow/example_dags/example_branch_operator.py took 0.043 seconds
